wallet/src/de/schildbach/wallet/WalletApplication.java

```diff
@@ -972,6 +980,7 @@ public void finalizeWipe() {
         shutdownAndDeleteWallet();
         cleanupFiles();
         config.clear();
+        clearDashPayPrefs();
```

Comment:
I don't think this is required here. `clearDatastorePrefs` below already removes all of datastore pref files.